azureproject/get_conn.py

```diff
@@ -0,0 +1,12 @@
+import os
+from flask import current_app
+from azure.identity import DefaultAzureCredential
+
+def get_conn():
+    # Azure hosted, refresh token that becomes password.
+    azure_credential = DefaultAzureCredential()
+    # Get token for Azure Database for PostgreSQL
+    print("Get password token.")
```

Comment:
Is print() needed? Doesnt seem very helpful. Would rather avoid print when possible if we dont needed.